template/vars/ub1804-gcp.yaml

```diff
@@ -0,0 +1,4 @@
+# This defines the current Taskcluster version we use for docker-worker, and force the V4L2LOOPBACK module to be build. It does not work on Ubuntu 18.04 GCP for Firefox CI if we don't build it.
```

Comment:
AIUI there is something that prevents us upgrading Ubuntu to 20.04 in some environments. It would be good to document this here, or link to a bug with more information.
Then next thing is that for some reason, we have to build V4L2Loopback under Ubuntu 18.04, rather than use a package, which it would also be good to document. This way it is clear that the reason we are building from source is because we have to on Ubuntu 18.04, and the reason we are using Ubuntu 18.04 is because of something else (which would also be documented here). This way, everyone can follow the argument, and have all the context they need to decide at some point in future if this is still valid or not. Thanks Michelle!

Reply:
I added some comments.